Sprint-3/revise/investigate/find.test.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
You missed implementing the function in this file.

Reply:
There isn't a function implementation need. There is already a function, and what I must do is use the Python Visualiser to help me play computer with this example, observe how this code is executed, and answering the questions